File: python/1116.py
q=int(input())
for x in range(1,q+1):
    x,y=[int(el) for el in input().strip().split(' ')]
    if y==0:
        print("divisao impossivel")
    else:
        c=x/y
        print('{:.1f}'.format(c))

File: javascript/1116.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var q=parseInt(lines.shift())

for(let i=1;i<=q;i++){
    [x,y]=lines.shift().trim().split(' ').map((el) => parseInt(el))
    if(y==0){
        console.log('divisao impossivel')
    }else{
        let c=x/y
        console.log(c.toFixed(1))
    }
    
}